modules/http/scanner.go

```diff
@@ -373,55 +382,82 @@ func getHTTPURL(https bool, host string, port uint16, endpoint string) string {
 
 // NewHTTPScan gets a new Scan instance for the given target
 func (scanner *Scanner) newHTTPScan(ctx context.Context, target *zgrab2.ScanTarget, useHTTPS bool, dialerGroup *zgrab2.DialerGroup) *scan {
+	cfg := &tls.Config{
+		InsecureSkipVerify: true,
+		ServerName:         "localhost",
```

Comment:
I am guessing that a ServerName MUST be supplied and we are doing this for IF we are upgraded to h2, and to ignore validation because that is what we want?